common/djangoapps/third_party_auth/saml.py

```diff
@@ -41,10 +41,18 @@ def auth_url(self):
         if not self._config.enabled:
             log.error('SAML authentication is not enabled')
             raise Http404
+
         # TODO: remove this check once the fix is merged upstream:
         # https://github.com/omab/python-social-auth/pull/821
```

Comment:
It would be an act of good citizenry, sure.  Happy to review the PR if you post one.